apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +19,35 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: undo removing book should add it list', async () => {
```

Comment:
It still does not pass :(

```
1) When: I use the reading list feature Then: undo removing book should add it list
  Message:
    Failed: element not interactable
```

Reply:
It was working when I pushed somehow I see error now. I have pushed few changes to e2e spec and now its detecting snack bar elements without any issue. 
Demo of e2e test run - https://recordit.co/FSdGNlt1py